src/screens/Search/SearchScreen/index.js

```diff
@@ -170,7 +170,7 @@ class SearchScreen extends PureComponent {
         style={styles.container}
         initialPage={(params && params.initialPage) || 0}
         renderTabBar={this.renderTabBar}
-        locked={this.state.isScrollingScene}
+        locked={Platform.OS === 'android'}
```

Comment:
Just a note that with latest RN there is a new `PlatformOS` which is a little leaner.